### DISCUSSION TITLE
OSClass future

### DISCUSSION THREAD

**Discussion by eurobank** [mindstellar/shopclass#483] [General]
Hello.

@navjottomer Can you please let us know what is the current state for Osclass? Will it be continued? Updated? Fixed?

Are there any issues in your development? Should we wait? Should we drop the project and look elsewhere?

**Comment by navjottomer** [comment]
Once I get enough funds in my pocket, last time I dedicated lots of time in
the project which nearly broke me. I am committed to this project will let
everyone know if I ever left it.
I have few current jobs which needs to be completed before I adjust my
schedule to get 5-10 hours a week on osclass.
For shopclass theme I am no more interested fixing that, it is better to
start from scratch for new theme. My website too need an update.

**Comment by eurobank** [comment]
Well nothing i can do about your financial status, i have donated personally a few months back but i guess nobody else did.

As for the rest you say, i completely understand but you must let us know of the current status. Otherwise you will loose the few people left here. We must know.

I do not like what i write but that is the hard truth. Nobody will trust a project that doesn't know where it is going. Great script, great developer, but nobody use it. I have seen that happen many-many times.

Again, i fully understand and also i know that almost nobody helps you. Still you must do something about it.

all the best.

**Comment by eurobank** [comment]
I guess i was right about asking for the status of the project on April 8. Now 6 moths later we have no information about anything. 

Bugs also remain unfixed. Pitty.

**Comment by navjottomer** [comment]
new update
